specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2020-04-01-preview/keyvault.json

```diff
@@ -1690,7 +1742,7 @@
           "type": "string",
           "description": "The reason for approval or rejection."
         },
-        "actionRequired": {
+        "actionsRequired": {
```

Comment:
Why plural if it's a single string? And should this be an enum? Need to document possible values.

Reply:
ActionsRequired is always "none" for keyVault. But other RPs use it for region expansion. For the plural concern, I don't have too much context, but seems like the parameter is also used by many other RPs. We may introduce this one into our code based on some previous requirements. So in our code it is always "ActionsRequired", here is should a typo bug we added before.

Reply:
The choice of `actionsRequired` is from network team. Every resource that supports private endpoints will describe this property with this exact naming.